test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -8,6 +8,10 @@
 // CHECK:   mutating func callAsFunction(_ x: Int32) -> Int32
 // CHECK:   mutating func callAsFunction(_ x: Int32, _ y: Int32) -> Int32
 // CHECK: }
+// CHECK: func == (lhs: LoadableIntWrapper, rhs: LoadableIntWrapper) -> Bool
```

Comment:
A nice side effect of this change is that `friend` operators now show up in the module interface.